### PR TITLE
Create Korean Localization strings.xml

### PR DESCRIPTION
**Pull request by umin135**
made Korean localization.

### CODE REVIEW

**Comment by a1batross** [comment]
Thank you for PR. But why it's half-Korean, half-English and even partially Russian? 

Don't include untranslatable strings, please. They are marked with `translatable="false"`.

**Comment by Wilson895** [comment]
> Thank you for PR. But why it's half-Korean, half-English and even partially Russian?
> 
> Don't include untranslatable strings, please. They are marked with `translatable="false"`.

This is incomplete. I'll ask you again after I fix it a little bit later.

**Comment by a1batross** [comment]
Oh, nice. :)

**Comment by a1batross** [comment]
You've missed a > in, after `"firstrun"`
```
 <string name="firstrun"설치 가이드 열기</string> 
```

File: res/values-ko/strings.xml [hunk 1]
```diff
@@ -45,51 +45,51 @@
...
+	<string name="text_shortcut_test">Xash3D FWGS 테스트 바로가기</string>
+	<string name="shortcut_button_save">바로가기 저장</string>
+	<string name="gamedir">모드 디렉토리</string>
+	<string name="pkgname">모드 패키지 이름 (숙련자만)</string>
+	<string name="shortcut_name">바로가기 이름</string>
 	<!-- Shorcut -->
 	
 	<!-- About -->
-	<string name="about_copyright">Flying With Gauss is not affiliated with Valve or any of their partners. All copyrights reserved to their respective owners.</string>
+	<string name="about_copyright">Fly With Gaus는 밸브 또는 해당 사업파트너와는 관련이 없습니다. 모든 저작권은 각각의 소유자에게 있습니다.</string>
```
Comment (lewa-j) [inline res/values-ko/strings.xml]: You have cut off half the name. And "Valve" too should not be translated.

**Comment by Wilson895** [comment]
Ah, ok

**Pull request closed without merging**